src/composables/useRenderer/index.ts

```diff
@@ -214,7 +218,7 @@ export function useRenderer(
       set(renderer.value, pathInThree, getValue(option, pathInThree))
 
     setValueOrDefault(options.shadows, 'shadowMap.enabled')
-    setValueOrDefault(options.toneMapping, 'toneMapping')
+    setValueOrDefault(options.toneMapping || ACESFilmicToneMapping, 'toneMapping')
```

Comment:
Is completely disallowing `NoToneMapping` the desired behavior here?

If not, it seems to me that this should use a nullish coalescing operator instead. E.g.,

```ts
setValueOrDefault(options.toneMapping ?? ACESFilmicToneMapping, 'toneMapping')
```

### Rationale

Logical OR (`||`) [returns the right side if the left side is falsy.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#description) `0` is falsy.

Three's [`NoToneMapping` constant has a value of `0`](https://github.com/mrdoob/three.js/blob/d04539a76736ff500cae883d6a38b3dd8643c548/src/constants.js#L54), i.e., falsy.

So ...

```ts
NoToneMapping || ACESFilmicToneMapping === ACESFilmicToneMapping // true
```